CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 
 ## Next
 
+## Unreleased
```

Comment:
This has the same meaning as `## Next` -> I'd delete this

Reply:
✅